src/primitives/plane.jl

```diff
@@ -60,4 +69,11 @@ normal(p::Plane) = normalize(p.u × p.v)
 
 ==(p₁::Plane{T}, p₂::Plane{T}) where {T} =
   isapprox((p₁.v - p₁.u) ⋅ normal(p₂), zero(T), atol=atol(T)) &&
-  isapprox((p₂.v - p₂.u) ⋅ normal(p₁), zero(T), atol=atol(T))
\ No newline at end of file
+  isapprox((p₂.v - p₂.u) ⋅ normal(p₁), zero(T), atol=atol(T))
+
+function Base.in(pt::Point, pl::Plane)
+  dotprod = normal(pl) ⋅ (pt - origin(pl))
+  T = typeof(dotprod)
+  isapprox(dotprod, zero(T); atol=atol(T))
```

Comment:
```suggestion
  d = normal(pl) ⋅ (pt - origin(pl))
  T = typeof(d)
  isapprox(d, zero(T), atol = atol(T))
```

Reply:
A comma to separate `atol`? Isn't it a keyword argument? In the help they always use a semi-colon.

Reply:
It is a matter of taste I guess. Only in specific cases the semi-colon makes a difference. So if we can stick to simple commas by default that would be more consistent with the rest of the codebase.